### PR TITLE
[release-4.8][manual] Backport podresources and podinfo commands

### PR DESCRIPTION
**Pull request by jlojosnegros**

### CODE REVIEW

File: pkg/controller/performanceprofile/components/tuned/tuned_test.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ var _ = Describe("Tuned", func() {
 					reservedSet, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
 					Expect(err).ToNot(HaveOccurred())
 					reserveCPUcount := reservedSet.Size()
-					channelsRegex := regexp.MustCompile(`\s*channels=combined ` + strconv.Itoa(reserveCPUcount) + `\s*`)
+					channelsRegex := regexp.MustCompile(`\s*channels=combined\s*` + strconv.Itoa(reserveCPUcount) + `\s*`)
```
Comment (fromanirh) [inline pkg/controller/performanceprofile/components/tuned/tuned_test.go:176]: this change should be reviewed separately, should not sneak in a unrelated backport

Comment (jlojosnegros) [reply pkg/controller/performanceprofile/components/tuned/tuned_test.go]: new PR #918

**Comment by jlojosnegros** [comment]
Depends on #918

**Review by marioferh — COMMENTED** [review]
/lgtm

**Comment by jlojosnegros** [comment]
rebase over #918

**Pull request merged**